prov/efa/src/rdm/efa_rdm_ep_progress.c

```diff
@@ -466,13 +466,14 @@ static inline void efa_rdm_ep_poll_ibv_cq(struct efa_rdm_ep *ep, size_t cqe_to_p
 		efa_rdm_tracepoint(poll_cq, (size_t) ep->ibv_cq_ex->wr_id);
 		if (ep->ibv_cq_ex->status) {
 			prov_errno = ibv_wc_read_vendor_err(ep->ibv_cq_ex);
-			if (ibv_wc_read_opcode(ep->ibv_cq_ex) == IBV_WC_SEND) {
+			opcode = ibv_wc_read_opcode(ep->ibv_cq_ex);
+			if (opcode == IBV_WC_SEND || opcode == IBV_WC_RDMA_WRITE) {
 #if ENABLE_DEBUG
 				ep->failed_send_comps++;
```

Comment:
we shouldn't mix send and write here. Add another failed_write_comps counter in ep and increment it here.